src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -204,6 +204,58 @@ public IActionResult BatchForecastData(IEnumerable<EmissionsForecastBatchDTO> re
         return Ok(result);
     }
 
+    /// <summary>
+    /// Retrieves the actual data that occurred between the time boundaries and calculates the average carbon intensity that occurred during that window. 
+    /// </summary>
+    /// <remarks>
+    ///  This endpoint is useful for reporting the measured carbon intensity for a specific time period in a specific location.
+    /// </remarks>
+    /// <param name="location"> the Azure location name of the region that we are measuring carbon usage in </param>
```

Comment:
Does it have to be Azure location? I thought we had code around supporting other location names eventually?

Reply:
Yeah, maybe one day ;) removed Azure